frontend/packages/vkt/src/interfaces/clerkEducation.ts

```diff
@@ -3,4 +3,14 @@ import { PublicFreeEnrollmentBasis } from 'interfaces/publicEducation';
 export interface ClerkFreeEnrollmentBasis extends PublicFreeEnrollmentBasis {
   comment: string;
   approved: boolean;
+  koskiEducations: KoskiEducations;
+}
+
+interface KoskiEducations {
+  matriculationExam: boolean;
+  higherEducationConcluded: boolean;
+  higherEducationEnrolled: boolean;
+  eb: boolean;
+  dia: boolean;
+  other: boolean;
 }
```

Comment:
```suggestion
type KoskiEducations = Record<EducationType, boolean>;
```

En kokeillut mutta tässä vois olla ehkä mahdollista tehdä näin (muualta koodista tosin pitää vaihtaa uppercase nuo)

Reply:
Joo, oishan toi mahdollista. Sitten tietty pitäisi joko tuolta bäkkäriltä suoraan palauttaa noita uppercase-versioita DTO:ssa tai serialisointivaiheessa muuttaa. Sinänsä mun puolesta käy kumpi tahansa vaihtoehto; ehkä hieman idiomaattisempaa olisi serialisointivaiheessa muuttaa, jos camelCase on ollut java-puolella käytössä?

Reply:
Hyvä pointti. Jätetään noin niinkuin teit alunperin 👍

Reply:
Ok.